src/Ansatz.jl

```diff
@@ -478,7 +478,7 @@ function simple_update_2site!(
 
     # truncate virtual index
     if any(!isnothing, (threshold, maxdim))
```

Comment:
If we just use `kwargs`, then you need to change it to...
```suggestion
    if any(!isnothing, get.(Ref(kwargs), [:threshold, :maxdim], nothing))
```